Makefile

```diff
@@ -644,6 +644,6 @@ dist/firecracker_kernel.tar.gz: firecracker/generate_firecracker_kernel.sh | dis
 FIRECRACKER_ROOTFS_FILES := $(shell find firecracker/packer -type f)
 dist/firecracker_rootfs.tar.gz: $(FIRECRACKER_ROOTFS_FILES) | dist
 	packer init -upgrade firecracker/packer/build-rootfs.pkr.hcl
-	packer build \
+	packer build -on-error=ask \
```

Comment:
How is this going to behave in CI if there's a failure? I don't think there's any way to get onto the builder to debug (at least, not how we're currently set up).

Reply:
WOOPS THIS IS NOT INTENTIONAL